AbletonMCP_Remote_Script/handlers/audio.py

```
@@ -0,0 +1,253 @@
+                "message": "Track is not frozen",
+            }
+
+        return {
+            "success": True,
+            "track_index": track_index,
+            "frozen": True,
+            "track_name": track.name,
+            "action_required": "manual_unfreeze",
+            "message": "Track unfreezing is not available via the Live Object Model API. "
+                       "Use Ableton's Edit menu or right-click the track header to unfreeze.",
+        }
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)